.github/workflows/rc-build.yml

```diff
@@ -0,0 +1,70 @@
+name: Release candidate build
```

Comment:
Should we name this `RC build` to match the other workflows?

Reply:
yeah can do